roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -267,7 +250,10 @@ def get_accepted_list(self):
             return np.where(self._accepted_list==1)[0].tolist()
 
     def get_rejected_list(self):
-        return [a for a in self.get_roi_ids() if a not in set(self.get_accepted_list())]
+        if self._rejected_list is None:
+            return [a for a in self.get_roi_ids() if a not in set(self.get_accepted_list())]
+        else:
```

Comment:
```suggestion
```
please remove this. Accepted is not the same thing as not rejected.

Reply:
fixed https://github.com/catalystneuro/roiextractors/pull/46/commits/e37c6092a6bfbef234d9ad1213e30ad2b780aa6a